boards/arm/stm32wb5mmg/stm32wb5mmg.dts

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2024 Javad Rahimipetroudi <javad.rahimipetroudi@mind.be>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/wb/stm32wb55Xg.dtsi>
+#include <st/wb/stm32wb55vgyx-pinctrl.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "STMicroelectronics STM32WB5MMG Bluetooth module";
+	compatible = "st,stm32wb5mmgh6";
+
+	chosen {
+		zephyr,console = &usart1;
+		zephyr,shell-uart = &usart1;
+		zephyr,bt-mon-uart = &lpuart1;
+		zephyr,bt-c2h-uart = &lpuart1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+	aliases {
+		watchdog0 = &iwdg;
+		die-temp0 = &die_temp;
+		volt-sensor0 = &vref;
+		volt-sensor1 = &vbat;
+	};
+};
+
+&die_temp {
+	status = "okay";
+};
+
+&clk_hse {
+	status = "okay";
+};
+
+&clk_lse {
+	status = "okay";
+};
+
+&clk_hsi48 {
+	status = "okay";
+};
```

Comment:
Do you actually need all these clocks ?

Reply:
For the `hsi48` when it is disabled, the following warning is reported. So, I put it in the DT:
```
ipm_stm32wb.c:510:2: warning: #warning Bluetooth IPM requires HSI48 clock to be enabled using device tree [-Wcpp]
  510 | #warning Bluetooth IPM requires HSI48 clock to be enabled using device tree
```

When `clk_lse` is disabled, the following error is reported:
```
ASSERTION FAIL [err == 0] @ WEST_TOPDIR/zephyr/subsys/bluetooth/host/hci_core.c:331
	command opcode 0x0c03 timeout with err -11
[00:00:10.000,000] <err> os: r0/a1:  0x00000003  r1/a2:  0x00000000  r2/a3:  0x00000002
[00:00:10.000,000] <err> os: r3/a4:  0x200005f8 r12/ip:  0x00002710 r14/lr:  0x0800335b
[00:00:10.000,000] <err> os:  xpsr:  0x49000000
[00:00:10.000,000] <err> os: Faulting instruction address (r15/pc): 0x08003366
[00:00:10.000,000] <err> os: >>> ZEPHYR FATAL ERROR 3: Kernel oops on CPU 0
[00:00:10.000,000] <err> os: Current thread: 0x200010d0 (unknown)
[00:00:10.056,000] <err> os: Halting system
```
Finally, when `clk_hse` is disabled, the following error is reported:
```
[00:00:00.000,000] <err> bt_driver: Unknown H:4 type 0x00
ASSERTION FAIL [err == 0] @ WEST_TOPDIR/zephyr/subsys/bluetooth/host/hci_core.c:331
	command opcode 0x0c03 timeout with err -11
[00:00:10.000,000] <err> os: r0/a1:  0x00000003  r1/a2:  0x00000000  r2/a3:  0x00000002
[00:00:10.000,000] <err> os: r3/a4:  0x200005f8 r12/ip:  0x00002710 r14/lr:  0x0800335b
[00:00:10.000,000] <err> os:  xpsr:  0x49000000
[00:00:10.000,000] <err> os: Faulting instruction address (r15/pc): 0x08003366
[00:00:10.000,000] <err> os: >>> ZEPHYR FATAL ERROR 3: Kernel oops on CPU 0
[00:00:10.000,000] <err> os: Current thread: 0x200010d0 (unknown)
[00:00:10.056,000] <err> os: Halting system

```

Reply:
Ok, let's keep them then ;). Thanks.